lib/IRGen/GenMeta.cpp

```diff
@@ -4201,7 +4206,13 @@ namespace {
              && "cannot generate metadata with placeholders in it");
     }
 
-    void addMethodOverride(SILDeclRef baseRef, SILDeclRef declRef) {}
+    void addMethodOverride(SILDeclRef baseRef, SILDeclRef declRef) {
+      for (auto &entry : VTableEntriesForVFE) {
```

Comment:
This change exists solely because I couldn't figure out how to go from a `SILFunction` pointed to in the VTable accessed in `addReifiedVTableEntry` to a corresponding `SILDeclRef`. Perhaps that is not possible, but if it is this is unnecessary.